post/templates/footer.html

```diff
@@ -0,0 +1,24 @@
+{% load i18n %}
+    <div class="footer-basic bg-dark">
```

Comment:
sai indent hỉ
`{% load i18n %}` -> thằng template tag này ko phải block nên ko cần lùi vào

các tag dạng block, có đóng mở thì phần nội dung code bên trong mới cần lùi vào. vd: `{block content} ... {%endblock%}`, `{%for ...%} ... {%endfor%}` ...

Reply:
dạ